config.go

```
@@ -135,6 +166,39 @@ func LoadConfig() (*Config, error) {
 		return nil, err
 	}
 
+	// Since our faraday directory overrides our TLS dir values, make sure
+	// that they are not set when loop dir is set. We hard here rather than
```

s/We hard/We fail hard?